src/processing/mode/android/AndroidMode.java

```diff
@@ -157,6 +157,20 @@ public void checkSDK(Editor parent) {
   public AndroidSDK getSDK() {
     return sdk;
   }
+  
+  
+  @Override
+  public String getSearchPath() {
+    StringBuilder tehPath = new StringBuilder(super.getSearchPath());
```

Comment:
No sense in using StringBuilder if you're just concatenating all the strings passed to it. Just do `return super.getSearchPath() + [everything else]`